### ISSUE TITLE
Material Theme UI crashes on Android Studio 3.2 Canary 10 and up

### ISSUE THREAD

**Issue by nanangarsyad** [ChrisRM/material-theme-jetbrains#732]
### Summary

Material UI theme unable to load on Android studio 3.2 Canary 10 and up

### Actual behavior

On startup if material UI theme in enabled it gave me this error messages and then by default Material UI theme got disabled.
![image](https://user-images.githubusercontent.com/7588442/39089098-63293002-45e9-11e8-9ea1-47edf7cb2427.png)

Here's the log (text) of above image in case needed

```
Plugin 'com.chrisrm.idea.MaterialThemeUI' failed to initialize and will be disabled.  Please restart Android Studio.

java.lang.NoClassDefFoundError: com/intellij/ide/ui/laf/darcula/ui/DarculaOptionButtonUI
    at com.chrisrm.idea.MTLafComponent.installDarculaDefaults(MTLafComponent.java:314)
    at com.chrisrm.idea.MTLafComponent.installDefaults(MTLafComponent.java:120)
    at com.chrisrm.idea.MTLafComponent.installMaterialComponents(MTLafComponent.java:77)
    at com.chrisrm.idea.MTLafComponent.initComponent(MTLafComponent.java:371)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:492)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:120)
    at com.intellij.openapi.application.impl.ApplicationImpl.lambda$createComponents$9(ApplicationImpl.java:434)
    at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:157)
    at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:580)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:441)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:104)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:393)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:379)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:198)
    at com.intellij.idea.MainImpl$1.lambda$null$0(MainImpl.java:49)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
    at java.awt.EventQueue.access$500(EventQueue.java:98)
    at java.awt.EventQueue$3.run(EventQueue.java:715)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:407)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.intellij.ide.ui.laf.darcula.ui.DarculaOptionButtonUI PluginClassLoader[com.chrisrm.idea.MaterialThemeUI, 2.1.7.1] com.intellij.ide.plugins.cl.PluginClassLoader@f6efa12
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:63)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    ... 33 more
```

### Steps to reproduce

1. Just open android studio preview 3.2 canary 10 and up, (3.2 canary 9 and bellow worked fine)

### Environment
From `Help -> about Intellij IDEA`
```
Android Studio 3.2 Canary 11
Build #AI-181.3007.14.32.4720098, built on April 14, 2018
JRE: 1.8.0_152-release-1136-b01 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.5-041505-generic

```
Material Theme Plugin version: v2.1.7
I also attached the `material_theme.xml` and `material_custom_theme.xml` but renamed it into `material_theme.txt` and `material_custom_theme.txt` becuase github not supporting that file type, Besides that, actually I'm not quite sure if those are the correct file. Please let me know

 if you need more information. 

Thank you.

[material_custom_theme.txt](https://github.com/ChrisRM/material-theme-jetbrains/files/1935039/material_custom_theme.txt)
[material_theme.txt](https://github.com/ChrisRM/material-theme-jetbrains/files/1935040/material_theme.txt)

**Comment by mallowigi** [comment]
geez, android studio didnt update the optionButton 😠

**Comment by mallowigi** [comment]
fixed in the last version 2.1.7.2.android

**Comment by bblori** [comment]
same issue in 2026, ubuntu 24, android studio 2025.1.3.7 , died after material theme install, please remove from market place.
